Lib/test/test_json/test_tool.py

```diff
@@ -96,6 +96,14 @@ def _create_infile(self, data=None):
             fp.write(data or self.data)
         return infile
 
+    def _create_outfile(self, existing_data=None):
```

Comment:
This function seems redundant.  See other tests which use outfile.

Reply:
Removed it.